### PR TITLE
Finance transaction hash and handle multiple chains

### PR DESCRIPTION
**Pull request by rperez89**

### CODE REVIEW

File: src/utils/web3-utils.js [hunk 1]
```diff
@@ -1,14 +1,27 @@
...
+  BLOCKSCOUT_TYPES,
+  ETHERSCAN_NETWORK_TYPES,
+  ETHERSCAN_TYPES,
+  POLYGONSCAN_NETWORK_TYPES,
+  POLYGONSCAN_TYPES,
+} from './provider-types'
 
 const DEFAULT_LOCAL_CHAIN = ''
+
 const networks = {
```
Comment (PJColombo) [inline src/utils/web3-utils.js:16]: Let's add mainnet as well:

```suggestion
const networks = {
mainnet: {
    chainId: 1,
    ensRegistry: '0x00000000000C2E074eC69A0dFb2997BA6C7d2e1e',
    name: 'Mainnet',
    type: 'mainnet',
    defaultEthNode: 'https://eth-rpc.gateway.pokt.network',
    explorer: 'etherscan',
    nativeToken: 'ETH',
  },
```

Check that the ens registry is correct.

File: src/utils/web3-utils.js [hunk 2]
```diff
@@ -1,14 +1,27 @@
...
+  ARBISCAN_TYPES,
+  BLOCKSCOUT_NETWORK_TYPES,
+  BLOCKSCOUT_TYPES,
+  ETHERSCAN_NETWORK_TYPES,
+  ETHERSCAN_TYPES,
+  POLYGONSCAN_NETWORK_TYPES,
+  POLYGONSCAN_TYPES,
+} from './provider-types'
 
 const DEFAULT_LOCAL_CHAIN = ''
```
Comment (PJColombo) [inline src/utils/web3-utils.js:14]: Should we add a default local chain :thinking: ?

**Review by PJColombo — COMMENTED** [review]
Hey @rperez89,

it seems an error is thrown whenever I click the `Transfer` button on the finance screen.

image

File: src/utils/web3-utils.js (hunk 2 above)
Comment (rperez89) [reply src/utils/web3-utils.js:14]: not sure, we don't have any local environment, what do you think?

Comment (PJColombo) [reply src/utils/web3-utils.js:14]: Let's leave it like this for now

**Pull request merged**